x/fundraising/types/msgs.go

```diff
@@ -277,6 +283,55 @@ func (msg MsgPlaceBid) GetBidder() sdk.AccAddress {
 	return addr
 }
 
+// NewModifyBid creates a new MsgModifyBid.
+func NewModifyBid(
+	auctionId uint64,
+	bidder string,
+	bidId uint64,
+	bidPrice sdk.Dec,
+	bidCoin sdk.Coin,
+) *MsgModifyBid {
+	return &MsgModifyBid{
+		AuctionId: auctionId,
+		Bidder:    bidder,
+		BidId:     bidId,
+		BidPrice:  bidPrice,
+		BidCoin:   bidCoin,
+	}
+}
+
+func (msg MsgModifyBid) Route() string { return RouterKey }
+
+func (msg MsgModifyBid) Type() string { return TypeMsgModifyBid }
+
+func (msg MsgModifyBid) ValidateBasic() error {
```

Comment:
Validity of MsgModifyBid
 - auction_id must be one of the existing auctions.
 - bid_id must be one of the existing bids in the auction with auction_id.
 - bid_price must be higher of the price of bid_id and/or coin amount must be higher of the coin amount of bid_id

Reply:
All those sanity checks must be done in keeper level. Let's handle those in this https://github.com/tendermint/fundraising/issues/93 issue later.